src/evaluations/data/evaluation_configs.py

```diff
@@ -966,19 +968,86 @@ def _generate_cardinality_estimator_configs():
   return tuple(configs)
 
 
+def _stratiefied_sketch_vector_of_counts(max_frequency, clip,
+                                         sketch_epsilon=None):
+  """Construct configs of StratifiedSketch based on VectorOfCounts.
+
+  Args:
+    max_frequency: an integer indicating the maximum frequency to estimate.
+    clip: a boolean indicating if or not to apply clipping for the
+      Vector-of-Counts sketch.
+    sketch_epsilon: the DP epsilon for noising the Vector-of-Counts sketch.
+
+  Returns:
+    A SketchEstimatorConfig for stratified sketch with Vector-of-Counts as its
+    base sketch.
+  """
+  if sketch_epsilon is not None:
+    sketch_epsilon_float = sketch_epsilon
+  else:
+    sketch_epsilon_float = float('inf')
+  sketch_operator = vector_of_counts_sketch_operator.StratifiedSketchOperator(
+      clip=clip,
+      epsilon=sketch_epsilon_float,
+  )
+  clip_str = 'clip' if clip else 'no_clip'
+  return SketchEstimatorConfig(
+      name=construct_sketch_estimator_config_name(
+          sketch_name='stratified_sketch_vector_of_counts',
+          sketch_config='4096',
+          estimator_name=f'sequential_{clip_str}',
+          sketch_epsilon=sketch_epsilon,
+          max_frequency=str(max_frequency)),
+      sketch_factory=stratified_sketch.StratifiedSketch.get_sketch_factory(
+          max_freq=max_frequency,
+          cardinality_sketch_factory=(
+              vector_of_counts.VectorOfCounts.get_sketch_factory(4096)
```

Comment:
I think 4096 should be an input to the config function.

Reply:
Added tests. Also added length as the argument.